pkgs/top-level/all-packages.nix

```diff
@@ -18861,4 +18861,12 @@ with pkgs;
   biblioteca = callPackage ../by-name/bi/biblioteca/package.nix {
     webkitgtk = webkitgtk_6_0;
   };
+
+  upbound = callPackage ../by-name/up/upbound/package.nix {
+    version-channel = "stable";
+  };
+
+  upbound-main = callPackage ../by-name/up/upbound/package.nix {
+    version-channel = "main";
+  };
```

Comment:
There is a merge conflict here.
You can avoid it by:
1. specifying version-channel to be "stable" by default
2. creating upbound-main in pkgs/by-name and overriding upbound there

Reply:
Done. Thank you for the feedback :)